components/gateway/pkg/proxy/adaptertransport.go

```diff
@@ -120,3 +225,15 @@ func (t *AdapterTransport) getClaims(headers http.Header) (Claims, error) {
 
 	return claims, nil
 }
+
+func min(a int, b int) int {
+	if a <= b {
+		return a
+	}
+
+	return b
+}
+
+func multipleOfFour(i int) int {
+	return ((i + 3) / 4) * 4
```

Comment:
The brackets around `(i + 3) / 4` are not actually needed.

Reply:
IMO it's clearer with the brackets